pilot/pkg/model/push_context.go

```diff
@@ -75,8 +75,8 @@ type PushContext struct {
 	// The following data is either a global index or used in the inbound path.
 	// Namespace specific views do not apply here.
 
-	// ServiceByHostname has all services, indexed by hostname.
-	ServiceByHostname map[Hostname]*Service `json:"-"`
+	// ServiceByHostnameAndNamespace has all services, indexed by hostname then namesace.
```

Comment:
typo. If you fix it: explain which namespace it is ( where the service was defined I assume )